clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -124,9 +124,9 @@
      */
     public static final String FETCH_MAX_BYTES_CONFIG = "fetch.max.bytes";
     private static final String FETCH_MAX_BYTES_DOC = "The maximum amount of data the server should return for a fetch request. " +
-            "This is not an absolute maximum, if the first message in the first non-empty partition of the fetch is larger than " +
-            "this value, the message will still be returned to ensure that the consumer can make progress. " +
-            "The maximum message size accepted by the broker is defined via <code>message.max.bytes</code> (broker config) or " +
+            "This is not an absolute maximum, if the first record batch in the first non-empty partition of the fetch is larger than " +
```

Comment:
I think this is the only place in the consumer documentation where we mention the concept of a record batch. I wonder if we should offer a brief explanation. For example: "Records are fetched in batches by the consumer. If the first record batch in the first non-empty.... "